src/main/frontend/components/publishing.cljs

```diff
@@ -0,0 +1,189 @@
+                              (notification/show! "Delete project successful." :success))))))
+          :background "pink")]
+
+       [:div.cp__publishing-pj-bt
+        (ui/button
+          (t :publishing/cancel)
+          :on-click (fn [e]
+                      (util/stop e)
+                      (reset! editor-state :display)))]])))
+
+(rum/defcs my-publishing
+  < rum/reactive db-mixins/query
+    (rum/local :display ::project-state)
+  [state]
+  (let [current-repo (state/sub :git/current-repo)
+        projects (state/sub [:me :projects])
+        pages (get-published-pages)
+        editor-state (get state ::project-state)
+        current-project (project-handler/get-current-project current-repo projects)]
+    (rum/with-context [[t] i18n/*tongue-context*]
+      [:div.flex-1
+       [:h1.title (t :my-publishing)]
+       [:div#cp__publishing-pj-ct
+        [:span (t :publishing/current-project)]
+        (project editor-state current-project pages)]
+       (when current-repo
+         [:div#cp__publishing-pg-ct
+          [:div (t :publishing/pages)]
+          [:table.table-auto
+                [:thead
+                 [:tr
+                  [:th (t :publishing/page-name)]
+                  [:th (t :publishing/delete-from-logseq)]]]
+                [:tbody
+                 (for [page pages]
+                   (let [page (first page)
+                         {:keys [title permalink]} (:page/properties page)
+                         page-name (:page/name page)]
+                     [:tr {:key permalink}
+                      [:td [:div.flex {}
+                            [:span [:a {:on-click (fn [e] (util/stop e))
+                                        :href (rfe/href :page {:name title})}
+                                    page-name]]
+                            [:span [:a {:href (util/format "%s/%s/%s" config/website current-project page-name)
+                                        :target "_blank"}
+                                    svg/external-link]]]
+                       ]
+                      [:td [:span.text-gray-500.text-sm
+                            [:a {:on-click
+                                 (fn [e]
+                                   (util/stop e)
+                                   (-> (p/let [_ (delete-page-from-logseq current-project permalink)]
+                                         (update-state-and-notify page-name))
+                                       (p/catch
+                                         (fn [error]
+                                           (let [status (.-status error)
+                                                 not-found-on-server 404]
+                                             (if (= not-found-on-server status)
+                                               (update-state-and-notify page-name)
+                                               (let [message (util/format "Remove Page \"%s\" from Logseq server failed."
```

Comment:
Consider to updating the message to "Failed to remove the page %s from Logseq.".